README.md

```diff
@@ -135,6 +136,7 @@ class Node extends libp2p {
         connEncryption: [
           SECIO
         ],
+        connProtector: new Protector(/*protector specific opts*/),
```

Comment:
Do we want to have multiple Protectors or just just have it inside and enable it as a config value?

Reply:
The implementation is based off of go which just supports a single, configurable protector. Users will be able to supply any protector they want, so it's extendable in the future as more protectors are created. In theory we could turn it into a list of protectors but I can't think of a use case where you'd want to do that as multiple levels of encryption on top of secio is really expensive. 

If we produce more "intelligent" protectors in the future, those would be easy to swap out.